sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveQl.scala

```diff
@@ -557,8 +557,11 @@ https://cwiki.apache.org/confluence/display/Hive/Enhanced+Aggregation%2C+Cube%2C
             "TOK_TABLEPROPERTIES"),
           children)
       val (db, tableName) = extractDbNameTableName(tableNameParts)
-
-      CreateTableAsSelect(db, tableName, nodeToPlan(query), allowExisting != None, Some(node))
+      if (ignores.exists(_ != None)) {
+        CreateTableAsSelect(db, tableName, nodeToPlan(query), allowExisting != None, Some(node))
+      } else {
+        CreateTableAsSelect(db, tableName, nodeToPlan(query), allowExisting != None, None)
```

Comment:
OK, I see. My concern of not relying on SemanticAnalyzer is for users who also have Hive installed, they may expect that the default format is controlled by `hive.default.fileformat`. If we pass `None` to `desc` at here, we will use text file instead of using the one specified in `hive.default.fileformat`.